generated/nixnet/nixnet.proto

```diff
@@ -1490,15 +1489,29 @@ message ReadSignalWaveformRequest {
     TimeOut timeout = 2;
     double timeout_raw = 3;
   }
-  uint32 size_of_value_buffer = 4;
+  uint32 number_of_samples = 4;
```

Comment:
nit: I might be misunderstanding this. "number_of_samples" is "samples_per_signal" right? Can we use that terminology instead (if that's accurate)? I'm concerned that a client might think that this is "total_number_of_samples" which is "number_of_samples * samples_per_signal".

Reply:
Yes, that makes much more sense, I'll change it everywhere.